### PR TITLE
Resolves #1939

### LINKED ISSUE #1939: Assign variable to value statement is wrong way around!
In this statement:

"The first command uses the New-ScheduledTaskAction cmdlet to assign the variable $A to the executable file tskmgr.exe."

assigning a variable to a value is not correct. In programming we assign a a value to a variable eg.

void example1 () {
    int a;
    a = 5;
}

The line "a = 5;" we would describe as "uses the equals sign to assign the value 5 to the variable a" as oppose to "uses the equals sign to assign the variable a to the value 5".

so the above statement should read 

"The first command uses the New-ScheduledTaskAction cmdlet to assign executable file tskmgr.exe to the variable $A."

This is one of several examples of this error on this page, and no doubt on several other pages as well.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0dfee6b1-12e6-67af-8a21-ed1bc2edeef3
* Version Independent ID: 5d00a090-0da0-0425-1845-cf833804a8ad
* Content: [New-ScheduledTask (scheduledtasks)](https://docs.microsoft.com/en-us/powershell/module/scheduledtasks/new-scheduledtask?view=win10-ps#feedback)
* Content Source: [docset/windows/scheduledtasks/New-ScheduledTask.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/scheduledtasks/New-ScheduledTask.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Reword example
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1939

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

cc

File: docset/winserver2012-ps/scheduledtasks/New-ScheduledTask.md [hunk 1]
```diff
@@ -163,13 +163,14 @@ Accept pipeline input: False
...
 
+### -Trigger
 ### -Trigger
 Specifies an array of one or more trigger objects that cause a scheduled task to start.
 
 A trigger is a set of criteria that starts a scheduled task when the criteria are met.
 You can use a time-based trigger or an event-based trigger to start a task, and one or more triggers can start a task.
 A task can have up to 48 triggers.
-For more information about triggers, see Triggershttp://technet.microsoft.com/en-us/library/cc748841.aspx.
+For more information about triggers, see [Triggers](https://technet.microsoft.com/en-us/library/cc748841.aspx).
```
Comment (get-itips) [inline docset/winserver2012-ps/scheduledtasks/New-ScheduledTask.md:173]:
```suggestion
For more information about triggers, see [Triggers](https://technet.microsoft.com/library/cc748841.aspx).
```

File: docset/winserver2012r2-ps/scheduledtasks/New-ScheduledTask.md [hunk 2]
```diff
@@ -182,7 +182,7 @@ Specifies an array of one or more trigger objects that cause a scheduled task to
 A trigger is a set of criteria that starts a scheduled task when the criteria are met.
 You can use a time-based trigger or an event-based trigger to start a task, and one or more triggers can start a task.
 A task can have up to 48 triggers.
-For more information about triggers, see [Triggers](http://technet.microsoft.com/en-us/library/cc748841.aspx).
+For more information about triggers, see [Triggers](https://technet.microsoft.com/en-us/library/cc748841.aspx).
```
Comment (get-itips) [inline docset/winserver2012r2-ps/scheduledtasks/New-ScheduledTask.md:185]:
```suggestion
For more information about triggers, see [Triggers](https://technet.microsoft.com/library/cc748841.aspx).
```

File: docset/windows/scheduledtasks/New-ScheduledTask.md [hunk 3]
```diff
@@ -183,7 +183,7 @@ Specifies an array of one or more trigger objects that cause a scheduled task to
 A trigger is a set of criteria that starts a scheduled task when the criteria are met.
 You can use a time-based trigger or an event-based trigger to start a task, and one or more triggers can start a task.
 A task can have up to 48 triggers.
-For more information about triggers, see [Triggers](http://technet.microsoft.com/en-us/library/cc748841.aspx).
+For more information about triggers, see [Triggers](https://technet.microsoft.com/en-us/library/cc748841.aspx).
```
Comment (get-itips) [inline docset/windows/scheduledtasks/New-ScheduledTask.md:186]:
```suggestion
For more information about triggers, see [Triggers](https://technet.microsoft.com/library/cc748841.aspx).
```

File: docset/winserver2012-ps/scheduledtasks/New-ScheduledTask.md (hunk 1 above)
Comment (velkovb) [reply docset/winserver2012-ps/scheduledtasks/New-ScheduledTask.md:173]: Good catch, I got the http but missed en-us :)

**Pull request merged**